README.md

```diff
@@ -1,43 +1,115 @@
 # Cloud Prototyping 2019/2020
 
-## Motivation
 
-This repository was created to automate the benchmarking process of ethereum with caliper. We have created the following docker images and automation scripts to make the process portable and repeatable as possible.
 
-## Docker
```

Comment:
It is fine to leave it but maybe in a later section

Reply:
But we decided to remove that folder of docker

Reply:
we can keep it is okay but just say it is if we want to run in docker for future work